modules/oxide-components/package.json

```diff
@@ -33,6 +33,7 @@
     "lint": "eslint --max-warnings=0 src/**/*.ts?",
     "start": "storybook dev -p 6006",
     "build-storybook": "storybook build",
+    "deploy-storybook": "gh-pages -d ./storybook-static -u 'tiny-bot <no-reply@tiny.cloud>' --nojekyll --dest modules/oxide-components",
```

Comment:
There should already be a CI username set through the Jenkins environment 🤔

Reply:
As Andy pointed out, CI already uses the `ephox` git user. You should be able to remove the `-u 'tiny...`` unless you really really want the gh-pages associated commits with that particular user

Reply:
Alright. I removed that user part 👍